[DEV-3197] Prevent graphql call from querying referrer when disabled (#269)
saasquatch/program-tools: packages/vanilla-components/CHANGELOG.md

# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

> ## [1.2.3] - 2022-03-01

Reviewer: Flagging date update needed

> ## [1.2.3] - 2022-03-01

Reviewer: ```suggestion
## [1.2.3] - 2022-03-02
```

> ## [1.2.3] - 2022-03-02

Author: ```suggestion
## [1.2.3] - 2022-03-03
```

> ## [1.2.3] - 2022-03-03

Reviewer: Update me!

### Updated

> - \<sqh-referral-list> showReferrer prop now prevents referrer from being queried when disabled

Author: ```suggestion
- \<sqh-referral-list> showReferrer prop now prevents referrer from being queried when set to "false"
```

> - \<sqh-referral-list> showReferrer prop now prevents referrer from being queried when set to "false"

Author: ```suggestion
- \<sqh-referral-list> show-referrer prop now prevents referrer from being queried when set to "false"
```
